nitro-protocol/contracts/MultiAssetHolder.sol

```diff
@@ -488,36 +404,10 @@ contract MultiAssetHolder is IMultiAssetHolder, StatusManager {
             keccak256(abi.encode(sourceOutcome))
         );
 
-        // store fingerprint of modified target outcome
-        targetOutcome[targetAssetIndex].allocations = newTargetAllocations;
-        _updateFingerprint(
-            targetChannelId,
-            claimArgs.targetStateHash,
-            keccak256(abi.encode(targetOutcome))
-        );
-
         // emit the information needed to compute the new source outcome stored in the fingerprint
-        emit AllocationUpdated(sourceChannelId, sourceAssetIndex, initialHoldings);
+        // emit AllocationUpdated(sourceChannelId, sourceAssetIndex); // TODO what do we really need here?
```

Comment:
Maybe we can add a separate event `Reclaimed` and use here for an intermediary to listen to.

Reply:
Good idea [38c1ccb](https://github.com/statechannels/go-nitro/pull/810/commits/38c1ccb2a49d63a7b128b0d024b75b4421f762ed). I think it is important for both parties in that ledger channel to listen to, because they won't be able to call claim a second time without computing the change in outcome themselves off chain (it is stored as a hash).